cves/CVE-2016-6795.yml

```diff
@@ -196,8 +196,8 @@ lessons:
     applies:
     note:
   complex_inputs:
-    applies:
-    note:
+    applies: true
+    note: URL input can be formatted in many ways and validating each different section of it correctly can be hard. In this case the validator did not do the job right in every case allowing attacker to traverse through the whole server
```

Comment:
I would add the pipe character "|" so you can split this message into multiple with newline characters to make it more readable.